compact/unused/ecal_barrel_sciglass_panda.xml

```diff
@@ -0,0 +1,82 @@
+<lccdd>
+  <comment>
+    Implementation of the projective barrel SciGlass calorimeter
+  </comment>
+
+  <define>
+    <constant name="EcalBarrel_Support_thickness" value="5*cm"/>
+
+    <documentation>
+      The EcalBarrel eta limits cut into the barrel shape.
+    </documentation>
+    <constant name="EcalBarrel_etamin" value="-3.5"/>
```

Comment:
I'm implementing the DIRC and wondering about this `etamin` value. This can't really be right, it would be 176 degrees in theta... We need to cut at a different `etamin` so there is room for the DIRC expansion boxes.

Reply:
In a case of 'design by wishful thinking' it seems to require about `etamin = -1.8` to avoid overlap with where the DIRC people think the DIRC expansion boxes go... In the old PANDA model that doesn't fit the last 4 rows of modules anymore (but I'm not that surprised by that).

Reply:
I also saw that the `etamin=-3.5` is not the acceptance for the current calorimeter and causes no cutout for the envelope if it is set according to the menagerie. The `-1.8` number sounds reasonable to me, we can confirm with GD/I later.

Reply:
Can you verify that that value does indeed not lead to overlaps of modules with the envelope?

Reply:
Yes, I will follow up on this